specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -106,11 +106,14 @@ model AnalyzeImageOptions {
 
 @doc("The image can be either base64 encoded bytes or a blob URL. You can choose only one of these options. If both are provided, the request will be refused. The maximum image size is 2048 x 2048 pixels and should not exceed 4 MB, while the minimum image size is 50 x 50 pixels.")
 @projectedName("csharp", "ContentSafetyImageData")
+@projectedName("java", "ContentSafetyImageData")
 model ImageData {
   @doc("The Base64 encoding of the image.")
   content?: bytes;
 
   @doc("The blob url of the image.")
+  @projectedName("csharp", "BlobUri")
+  @projectedName("java", "blobUri")
```

Comment:
I am not sure if Java would call it `uri`?

Reply:
keep bloburl